src/web.cpp

```diff
@@ -1077,7 +1077,8 @@ void handle_firmware_upload()
         {
             uint32_t maxSketchSpace = (ESP.getFreeSketchSpace() - 0x1000) & 0xFFFFF000;
```

Comment:
getFreeSketchSpace already starts at the first free erase sector.  Why was this math added?

Reply:
I think I copied from somewhere else.  But now that I have looked at how the Updater.begin() code is written it is unnecessary.